sdk/typescript/runtime/main.go

```diff
@@ -12,11 +12,11 @@ import (
 )
 
 const (
-	bunVersion  = "1.0.27"
+	bunVersion  = "1.1.12"
 	nodeVersion = "21.3"
 
 	nodeImageDigest = "sha256:3dab5cc219983a5f1904d285081cceffc9d181e64bed2a4a18855d2d62c64ccb"
-	bunImageDigest  = "sha256:82d3d3b8ad96c4eea45c88167ce46e7e24afc726897d48e48cc6d6bf230c061c"
+	bunImageDigest  = "sha256:6568a679b87107d3d7d46b829f614c443e73bbe3bf7d6ea5c9ceb8f845869c96"
```

Comment:
Hm, this has happened before as well.

I wonder if we need some more tests where we *don't* have emulation enabled to be able to catch these.

Reply:
The thing is, I did have emulation enabled (`docker run --privileged --rm tonistiigi/binfmt --install all`) and this still failed. Mildly confused as to why... but I think the only way to catch this robustly is to just have ARM runners in addition to x86.